src/components/app.cjsx

```diff
@@ -97,18 +116,22 @@ module.exports = React.createClass
         <div className="container-fluid">
           <div className="navbar-header">
             <BS.ButtonToolbar className="navbar-btn">
-              <a href="/exercises/new" className="btn btn-success">New Blank Exercise</a>
-              <AsyncButton
-                bsStyle='info'
-                onClick={@saveExercise}
-                disabled={isWorking}
-                isWaiting={ExerciseStore.isSaving(id)}
-                waitingText='Saving...'
-                isFailed={ExerciseStore.isFailed(id)}
-                >
-                Save Draft
-              </AsyncButton>
-              { if not ExerciseStore.isNew(id)
+              <a href="/exercises" onClick={@onReset} className="btn btn-danger">Reset</a>
```

Comment:
Hm, I don't understand your meaning, sorry. 

`a` tags don't have an `onpageload` event.  Are you suggesting that `onReset` be called `onpageload` ?  I don't think that'll work because react hasn't rendered yet for that.